### PR TITLE
Added Presence • Pulse 🎶

### PR DESCRIPTION
**Pull request by callumok2004**
https://cdn.discordapp.com/attachments/630142339872194591/680164993177878532/unknown.png

### CODE REVIEW

File: Pulse/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "id": "506899274748133376"
+    },
+    "url": "itspulse.net",
+    "description": {
+      "en": "Pulse is an internet radio community originally started in 2019, after much disrespect and un-professional antics across the radio community; we decided to make our own project that gives people the chance and opportunity to take control of how they want a community."
+    },
+    "service": "Pulse",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/mqYHGOC.png",
+    "thumbnail": "https://cdn.discordapp.com/attachments/630142339872194591/680154520919081024/unknown.png",
```
Comment (frozaum) [inline Pulse/dist/metadata.json]: Please reupload the image to some Image file hosting service, thanks.

Comment (Bas950) [inline Pulse/dist/metadata.json]: upload thumbnail to imgur

**Comment by callumok2004** [comment]
Changed to imgur

File: Pulse/dist/metadata.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "service": "Pulse",
     "version": "1.0.0",
     "logo": "https://i.imgur.com/mqYHGOC.png",
-    "thumbnail": "https://cdn.discordapp.com/attachments/630142339872194591/680154520919081024/unknown.png",
+    "thumbnail": "https://i.imgur.com/eeZ8Wlm.png",
```
Comment (frozaum) [inline Pulse/dist/metadata.json]: uh
> The media you include (icon/thumbnail) must be related to the website and should be understandable in terms of size and quality.

Comment (callumok2004) [reply Pulse/dist/metadata.json]: ofc I am the stupid idiot that used the wrong link HAHA

**Pull request merged**